src/index.js

```diff
@@ -1,5 +1,5 @@
-import redata from './redata';
+import configRedata from './redata';
 import reactRedata from './react/react-redata';
 
-export default redata;
+export default configRedata;
```

Comment:
@antoniocapelo did right in maintaining consistency.

As for ambiguous `redata` keywords, I agree that the concept has been evolving and, right now, I consider a `redata` a *configured redata* or, for the lack of a better term, a *redata instance*. That's what you get when you call `configRedata`. I've been considering renaming it to `createRedata`, which would need to be changed in both places, like @antoniocapelo did. I'll have a look overall again after the composition changes that I'm implementing right now, clean up the concepts and then we discuss if everyone's happy.

Overall, we need to keep in mind that redata is one thing, react-redata is another. Even though it started as react driven, it is and should be agnostic. I believe that in time we might even want to move into a separate repository, and have react users install `react-redata` instead of `redata`.